starter-code/pizza.js

```diff
@@ -1 +1,170 @@
 // Write your Pizza Builder JavaScript in this file.
+//   if ($(".btn-pepperonni").hasClass("active")) {
+//     // $("section[class^='pep ']").css("display", "none");
+//     // $(".pep").hide();
+//     $(".btn-pepperonni").removeClass("active");
+//   } else {
+//     // $(".pep").show();
+//     $(".btn-pepperonni").addClass("active");
+//   }
+
+var price = 13;
+// var price =
+//   parseInt(
+//     $("aside b")
+//       .html()
+//       .substr(1)
+//   ) +
+//   parseInt(
+//     $("aside ul li:nth-child(1)")
+//       .html()
+//       .substr(1)
+//   ) +
+//   parseInt(
+//     $("aside ul li:nth-child(2)")
+//       .html()
+//       .substr(1)
+//   ) +
+//   parseInt(
+//     $("aside ul li:nth-child(3)")
+//       .html()
+//       .substr(1)
+//   ) +
+//   parseInt(
+//     $("aside ul li:nth-child(4)")
+//       .html()
+//       .substr(1)
+//   ) +
+//   parseInt(
+//     $("aside ul li:nth-child(5)")
+//       .html()
+//       .substr(1)
+//   );
+
+$(".btn-pepperonni").click(function() {
+  $(".pep").toggle();
+  $(".btn-pepperonni").toggleClass("active");
+  $("aside ul li:nth-child(1)").toggle();
+
+  if ($(".btn-pepperonni").hasClass("active")) {
+    // $("aside ul li:nth-child(1)").show();
+    price += parseInt(
```

Comment:
Creative solution :)
Another way is to hold the logic separately, in an object, something like 
`var priceList =
{
pepperoni : 1,
mushroom : 1,
...
...
etc....
}`

Then you could just initiate the price at 0 and do something like 
`
price += priceList.pepperoni
`